Traceback (most recent call last):
  File "/Users/dhgajend/my_home/workspace/cloud/logger/temp/job4/tmp/gen_script.py", line 20, in <module>
    inp_script.execute(__data_bag, cpsLogger)
TypeError: execute() takes exactly 1 argument (2 given)
